src/elements/lead-container/lead-container.visual.spec.ts

```diff
@@ -31,6 +32,15 @@ describe(`sbb-lead-container`, () => {
         }
       </style>
       ${image}
+      <sbb-alert-group class="sbb-lead-container-spacing">
```

Comment:
I remember that Florian told me, the alert group should be after the breadcrumb. Did you discuss this?

Reply:
I checked it with florian. it is OK as it is.